tools/docker-copy.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+INPUTS=("${@}")
+TARGET_ARCH=${TARGET_ARCH:-amd64}
+DOCKER_WORKING_DIR=${INPUTS[${#INPUTS[@]}-1]}
+FILES=${INPUTS[@]:0:${#INPUTS[@]}-1}
+
+function may_copy_into_arch_named_sub_dir() {
+  FILE=${1}
+  COPY_ARCH_RELATED=${COPY_ARCH_RELATED:-1}
+
+  FILE_INFO=$(file "${FILE}")
+  # when file is an executable, like binary,
+  # will put an arch named sub dir
+  # like
+  #   arm64/
+  #   amd64/
+  if [[ ${FILE_INFO} == *"executable"* ]]; then
+    chmod 755 "${FILE}"
```

Comment:
`executable` will be a binary.

then `chmod here`, so `chmod 755  *` in Makefiles removed